### PR TITLE
#571 - Added included path patterns support to RepoPermissions.update

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #571 
Added included path patterns support to `RepoPermissions#update`

### CODE REVIEW

File: src/main/java/com/artipie/RepoPermissions.java [hunk 1]
```diff
@@ -66,12 +67,17 @@
...
      * @param repo Repository name
      * @param permissions Permissions list
+     * @param patterns Included path patterns
      * @return Completion action
      */
-    CompletionStage<Void> addUpdate(String repo, Collection<UserPermission> permissions);
+    CompletionStage<Void> update(
+        String repo,
+        Collection<UserPermission> permissions,
+        List<String> patterns
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/RepoPermissions.java:79]: @olegmoz I'd suggest use `Collection` here instead of list

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz just one suggestion, please, take a look

File: src/main/java/com/artipie/RepoPermissions.java (hunk 1 above)
Comment (olegmoz) [reply src/main/java/com/artipie/RepoPermissions.java]: @olenagerasimova thanks, you are right. Fixed it

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#598` is not assigned, can't get performer

**Pull request merged**